Eatery/Controllers/BRB/BRBViewController.swift

```diff
@@ -15,26 +16,41 @@ class BRBViewController: UIViewController {
         case account(BRBAccount)
     }
     
-    private var connectionHandler: BRBConnectionHandler!
+    var accountManager: BRBAccountManager!
     private var requestStart: Date?
+    private var account: BRBAccount?
+    private var loggedIn = false
     
     private lazy var loginViewController = BRBLoginViewController()
-    private var accountViewController: BRBAccountViewController?
+    var accountViewController: BRBAccountViewController?
```

Comment:
why do we need to make this `public`?

Reply:
It should be private. In a previous commit it was being accessed from the EateryTabBarController, but no longer is.